src/views/v2/tickets/item/content.php

```diff
@@ -8,21 +8,36 @@
  *
  * See more documentation about our views templating system.
  *
- * @link http://m.tri.be/1amp
+ * @link    http://m.tri.be/1amp
+ *
+ * @since   TBD
  *
- * @since TBD
  * @version TBD
  *
- * @var Tribe__Tickets__Ticket_Object   $ticket Ticket Object.
- * @var int                             $key Ticket Item index
+ * @var Tribe__Tickets__Editor__Template   $this            The template instance.
+ * @var Tribe__Tickets__Ticket_Object      $ticket          Ticket Object.
+ * @var int                                $key             Ticket Item index.
+ * @var string                             $content         Message.
+ * @var Tribe__Tickets__Commerce__Currency $currency        The Currency Object.
+ * @var string                             $currency_symbol The currency symbol, e.g. '$'.
+ * @var int                                $key             Ticket Item index.
+ * @var WP_Post|int                        $post_id         The post object or ID.
+ * @var Tribe__Tickets__Tickets            $provider        The tickets provider class.
+ * @var bool                               $is_mini         True if it's in mini cart context.
  */
 
 if ( empty( $ticket ) ) {
 	return;
 }
 
-$this->template( 'v2/tickets/item/content/title', [ 'ticket' => $ticket, 'key' => $key ] );
+$context = [
+	'ticket'      => $ticket,
+	'key'         => $key,
+	'provider_id' => $provider->class_name,
```

Comment:
Confirm that provider_id is not already passed, but you can remove ticket/key from this $context array since they will be inherited.

Reply:
fixed